pkg/cloudprovider/types.go

```diff
@@ -41,6 +42,16 @@ var (
 
 type DriftReason string
 
+type RepairStatement struct {
+	// Type of unhealthy state that is found on the node
+	Type corev1.NodeConditionType
```

Comment:
Might be nice to call it ConditionType so it hints to the mechanism it relies on?
```suggestion
	ConditionType corev1.NodeConditionType
```